src/canvas-extractor/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "edfi-canvas-extractor"
-version = "1.1.0"
+version = "1.2.0"
```

Comment:
I bumped application minor versions in the hope that we do release some performance improvements, and so that we would have version number consistency.

But for libraries, you'll see that I just bumped the patch version.